### PR TITLE
Update to AM 8.1.0

### PR DESCRIPTION
**Pull request by Tasteful**
This is an update to use AM 8.1.0 with the new "features"-feature to store the AM.Collection configuration.

Also refactor the test to run in parallel.

Fixing: #109 #120

### CODE REVIEW

**Comment by Tasteful** [comment]
@TylerCarlson1 will you take a look at this PR also, thanks.

**Comment by TylerCarlson1** [comment]
I'm looking at it, so far so good it's just I'm still trying to navigate it all, so it will take some more time to completely look over it all.

File: src/AutoMapper.Collection/Configuration/GeneratePropertyMapsExpressionFeature.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper.Collection.Runtime;
+using AutoMapper.EquivalencyExpression;
+using AutoMapper.Features;
+
+namespace AutoMapper.Collection.Configuration
+{
+    public class GeneratePropertyMapsnFeature : IGlobalFeature
```
Comment (TylerCarlson1) [inline src/AutoMapper.Collection/Configuration/GeneratePropertyMapsExpressionFeature.cs]: GeneratePropertyMapsExpressionFeature suppose to be name?

Comment (Tasteful) [reply src/AutoMapper.Collection/Configuration/GeneratePropertyMapsExpressionFeature.cs]: Yes, that is correct, fixed.

**Comment by Tasteful** [comment]
I can see that the `WIP — Please accept the new permissions` is failing, @TylerCarlson1 or @jbogard can any of you look into that one?

**Comment by TylerCarlson1** [comment]
I couldn't stop the WIP but it looked good so just merged it.

**Pull request merged**